NetCoreRestApi/WebAPI/Controllers/ProductsController.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using BusinessLayer.Manager;
+using Microsoft.AspNetCore.Mvc;
+using ServiceLayer.Converters;
+using ServiceLayer.DataTransferObjects;
+
+namespace WebAPI.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ProductsController : ControllerBase
+    {
+        [HttpGet]
+        public ActionResult<IEnumerable<ProductDTO>> Get()
+        {
+            ProductManager productManager = new ProductManager();
```

Comment:
wrong initialization of productManager. it should be initialized over DI . Put IProductManager interface as private field. Inject interface into controller constuctor.

Reply:
Done